src/GraphExplorer.js

```diff
@@ -0,0 +1,313 @@
+/**
+ * Created by valeriy.abornyev on 11/9/2015.
+ */
+
+var vis = require('vis'),
+    $ = require('jquery'),
+    ElementsCollection = require('./ElementsCollection');
+
+
+    function GraphExplorer(settings) {
```

Comment:
The general problem with your constructor is that it's to fat (a lot of code lines) so 
reader can't understand what's you are doing from the hight point of view

>Hint: try to review it precisely and refactor it by moving some code parts to methods